include/config.h

```diff
@@ -102,10 +104,22 @@ struct ConfigStruct {
 
   // Vibration Output Mode
   VibrationMode vibration_mode;
+  
+  // Post orgasm torure stuff
+  int clench_pressure_sensitivity;
+  int max_clench_duration = 100; 
```

Comment:
I plan to cross compile this structure into C code, so this default value won't do. Move defaults into the JSON loader in Config.cpp with the others, and only declare variables in this struct but not initialize them.

This is also how the static code analysis checks for defaults, which you may have already done because I see it passed.

Reply:
i integrated in the config but not presenting this in the GUI. There should not be reason to change the values. The variable is still modifiable on the terminal if really needed